src/Sylius/Bundle/ApiBundle/Resources/config/services/context_builders.xml

```diff
@@ -78,7 +78,8 @@
         </service>
 
         <service
-            id="Sylius\Bundle\ApiBundle\SerializerContextBuilder\LoggedInShopUserIdAwareContextBuilder"
+            id="sylius_api.serializer_context_builder.logged_in_shop_user_id_aware"
```

Comment:
I would be for:
```suggestion
            id="sylius_api.serializer.context_builder.logged_in_shop_user_id_aware"
```
or
```suggestion
            id="sylius_api.context_builder.logged_in_shop_user_id_aware"
```
but we could adjust that after refactoring the directory structure